Evaluation/Input/Content/Section2.1_Modeling_Strategy.md

```
@@ -0,0 +1,20 @@
+The general concept of building a PBPK model has previously been described by Kuepfer et al. ([Kuepfer 2016](#main-references)) The relevant anthropometric (height, weight) and physiological parameters (e.g. blood flows, organ volumes, binding protein concentrations, hematocrit, cardiac output) in adults was gathered from the literature and has been previously published ([PK-Sim Ontogeny Database Version 7.3](#main-references)). The information was incorporated into PK-Sim® and was used as default values for the simulations in adults.
+
+The applied activity and variability of plasma proteins and active processes that are integrated into PK-Sim® are described in the publicly available PK-Sim® Ontogeny Database Version 7.3 ([Schlender 2016](#main-references)) or otherwise referenced for the specific process.
+
+The present model represents an updated version of the model presented by Cicali et al. ([Cicali 2021](#main-references)). A PBPK model for levonogestrel was developed and evaluated for healthy adult volunteers for IV and oral administration. The mean PBPK model was developed using a standard female individual. The model was evaluated by comparing the pharmacokinetic simulation with observed in vivo pharmacokinetic data taken from literature. The current levonogestrel model is based on the publication by Cicali et al [Cicali 2021](#main-references).
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)